tests/rptest/clients/rpk.py

```diff
@@ -1472,3 +1479,54 @@ def delete_subjects(self, subjects=[], permanent=False):
             cmd += ["--permanent"]
 
         return self._run_registry(cmd)
+
+    def _run_wasm(self, rest):
+        cmd = [self._rpk_binary(), "transform"]
+        cmd += ["-X", "admin.hosts=" + self._redpanda.admin_endpoints()]
+        cmd += rest
+        return self._execute(cmd)
+
+    def deploy_wasm(self, name, input_topic, output_topic):
+        self._run_wasm([
```

Comment:
Do we not expect any output from this command?

Reply:
Nope. Just exitcode 0 or 1 and `_execute` will throw if the exitcode is not 0.